test/net/sourceforge/kolmafia/textui/RuntimeLibraryTest.java

```diff
@@ -650,4 +652,74 @@ void canIdentifySkill(final String skillIdentifier) {
     String output = execute("$skill[" + skillIdentifier + "].name");
     assertThat(output, endsWith("Returned: [zero placeholder]\n"));
   }
+
+  @Nested
+  class EightBitPoints {
+    @Test
+    void zeroPointsInNon8BitZone() {
+      String output = execute("eight_bit_points($location[The Dire Warren])");
+      assertThat(output, endsWith("Returned: 0\n"));
+    }
+
+    @ParameterizedTest
+    @CsvSource({"blue, 60", "black, 30"})
+    void fungusPlains(String color, int points) {
+      var cleanups =
+          new Cleanups(
+              withEquipped(ItemPool.CARPE), // 50% meat drop
+              withEquipped(ItemPool.ORIGAMI_PASTIES), // 30% meat drop
+              withEquipped(ItemPool.PANTSGIVING), // 30% meat drop
+              withProperty("8BitColor", color));
+
+      try (cleanups) {
+        String output = execute("eight_bit_points($location[Fungus Plains])");
+        assertThat(output, endsWith("Returned: " + points + ".0\n"));
+      }
+    }
+
+    @ParameterizedTest
+    @CsvSource({"green, 30", "black, 15"})
+    void herosField(String color, int points) {
+      var cleanups =
+          new Cleanups(
+              withEquipped(ItemPool.GRIMACITE_GO_GO_BOOTS), // 30% item drop
+              withProperty("8BitColor", color));
+
```

Comment:
30% item drop in a zone that gives max bonus at 400%.
Base points are 50 or 100.